### ISSUE TITLE
8.4.1/3.0.3 login hostname not updating

### ISSUE THREAD

**Issue by themorey** [Azure/cyclecloud-slurm#156]
CC ver = 8.4.1
Slurm project ver = 3.0.3

**ISSUE**:
Login nodes do not update hostname to `cluster-prefix` value defined in cluster when `use_node_as_hostname` is enabled.  In same cluster the scheduler and execute node hostnames are updated as expected.

**INFO**:
```
jmorey@8fgxl000001:~$ sudo -i jetpack config hostname
8fgxl000001

jmorey@8fgxl000001:~$ sudo -i jetpack config slurm.node_prefix
jm-cryo
```

**EXPECTED BEHAVIOR**:
hostname should be set to `jm-cryo-login-1` not `8fgxl000001`

**Comment by themorey** [comment]
the [install.py](https://github.com/Azure/cyclecloud-slurm/blob/11a29609275dc1676749411cf5a8f8b3738193c5/slurm/install/install.py#L475)  file calls the `set_hostname` function for execute nodes.  Should it also call it for login?

```
    if args.mode == "execute":
        set_hostname(settings)
        setup_slurmd(settings)
```

**Comment by aditigaur4** [comment]
Yes we should do that. Ryan has a fix WIP for this.

**Comment by ryanhamel** [comment]
Closed in previous login fix, 4e28b1e83a117d046a40b81780226f6d14ca91f3
